kubectl-minio/cmd/helpers/helpers.go

```diff
@@ -148,3 +166,43 @@ func ToYaml(objs []runtime.Object) ([]string, error) {
 
 	return manifests, nil
 }
+
+// GetTable returns a formatted instance of the table
+func GetTable() *table.Table {
+	t := table.NewWriter(os.Stdout)
+	t.SetAutoWrapText(false)
+	t.SetHeaderAlignment(table.ALIGN_LEFT)
+	t.SetAlignment(table.ALIGN_LEFT)
+	return t
+}
+
+// DisableHelp disables the help command
+func DisableHelp(cmd *cobra.Command) *cobra.Command {
+	cmd.SetHelpCommand(&cobra.Command{
+		Use:    "no-help",
+		Hidden: true,
+	})
+	return cmd
+}
+
+// Ask user for Y/N input. Return true if response is "y"
+func Ask() bool {
+	reader := bufio.NewReader(os.Stdin)
+	for {
+		s, _ := reader.ReadString('\n')
+		s = strings.TrimSuffix(s, "\n")
+		s = strings.ToLower(s)
+		if len(s) > 1 {
+			fmt.Fprintln(os.Stderr, "Please enter Y or N")
+			continue
+		}
+		if strings.Compare(s, "n") == 0 {
+			return false
+		} else if strings.Compare(s, "y") == 0 {
+			break
+		} else {
+			continue
+		}
+	}
+	return true
+}
```

Comment:
You may want to use https://github.com/manifoldco/promptui

Reply:
```go
package main

import (
        "fmt"

        "github.com/manifoldco/promptui"
)

func main() {
        prompt := promptui.Prompt{
                Label:     "Delete Resource",
                IsConfirm: true,
        }

        result, err := prompt.Run()

        if err != nil {
                fmt.Printf("Prompt failed %v\n", err)
                return
        }

        fmt.Printf("You choose %q\n", result)
}
```

Reply:
done